lib/crit_tables/slash_critical_table.rb

```diff
@@ -87,7 +87,7 @@ module CritRanks
                                   :slowed          => false,
                                   :wound_rank      => 2,
                                   :secondary_wound => nil,
-                                  :regex           => /(?!.*removes skull.)Blow to head\./ },
+                                  :regex           => /(?!.*removes skull.)Blow to head(?:\.|\!)/ },
```

Comment:
Preferred to have 'any one character' regex (`.`) over explicitly declaring punctuation.  This may be a problem with the initial submit - an actual period rather than a 1-char regex.  Can this be reverted to that format as the correction?

`/(?!.*removes skull.)Blow to head./ },`